__tests__/app.test.js

```diff
@@ -110,4 +110,33 @@ describe("GET /api/articles", () => {
                 expect(articles).toBeSortedBy('created_at', { descending: true })
             })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: should respond with an array of comments for the given article_id ", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toBeInstanceOf(Array);
+                expect(comments).toHaveLength(11);
+                comments.forEach(comment => {
+                    expect(comment).toHaveProperty("comment_id", expect.any(Number));
+                    expect(comment).toHaveProperty("votes", expect.any(Number));
+                    expect(comment).toHaveProperty("created_at", expect.any(String));
+                    expect(comment).toHaveProperty("author", expect.any(String));
+                    expect(comment).toHaveProperty("body", expect.any(String));
+                    expect(comment).toHaveProperty("article_id", expect.any(Number));
+                })
+            })
+    })
+    test("400: Error - should return bad request when passing invalid article_id", () => {
+        return request(app)
+            .get('/api/articles/car/comments')
+            .expect(400)
+            .then(({ body }) => {
+                expect(body).toEqual({ msg: "Bad Request" });
+            })
```

Comment:
You should also test for a valid ID that exxists but has no comments. This should return an empty array.

Reply:
Added a test for this case